packages/conf-graphics/conf-graphics.1.0/opam

```diff
@@ -0,0 +1,8 @@
+opam-version: "1"
+maintainer: "Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"
+homepage: "http://www.ocaml.org/"
+license: "LGPL+EXN-BIN"
+build: [["ocamlc" "-custom" "graphics.cma" "-o" "test"]]
+post-messages: [
+  "This package checks whether the Graphics library was compiled." {failure}
```

Comment:
Unfortunately, I have no such hint: I only use Debian, and even on it, the ./configure of OCaml fails to detect X11 dev files on some versions. Anyway, the Graphics library will probably be moved out of OCaml soon, as many of the other libraries, so it is a temporary problem.